_utility-aliases.scss

```diff
@@ -55,77 +55,85 @@ $text-color--positive: $color--fa-green-text;
 
 /* Color map */
 $color-values: (
-  alt:           $color--alt,
-  base:          $color--base,
-  fa-amber:      $color--fa-amber,
-  fa-blue:       $color--fa-blue,
-  fa-blue-dark:  $color--fa-blue-dark,
-  fa-blue-light: $color--fa-blue-light,
-  fa-cyan:       $color--fa-cyan,
-  fa-green:      $color--fa-green,
-  fa-green-dark: $color--fa-green-dark,
-  fa-orange:     $color--fa-orange,
-  fa-pink:       $color--fa-pink,
-  fa-purple:     $color--fa-purple,
-  fa-red:        $color--fa-red,
-  fa-red-orange: $color--fa-red-orange,
-  fa-yellow:     $color--fa-yellow,
-  gray-1:        $color--gray-1,
-  gray-2:        $color--gray-2,
-  gray-3:        $color--gray-3,
-  gray-4:        $color--gray-4,
-  gray-5:        $color--gray-5,
-  gray-6:        $color--gray-6,
-  gray-7:        $color--gray-7,
-  gray-8:        $color--gray-8,
-  gray-9:        $color--gray-9,
-  gray-10:       $color--gray-10,
-  gray-11:       $color--gray-11,
-  gray-12:       $color--gray-12,
-  gray-13:       $color--gray-13,
-  gray-14:       $color--gray-14,
-  gray-15:       $color--gray-15,
-  gray-16:       $color--gray-16
+  alt:                    $color--alt,
+  base:                   $color--base,
+  fa-amber:               $color--fa-amber,
+  fa-blue:                $color--fa-blue,
+  fa-blue--shade-1:       $color--fa-blue--shade-1,
```

Comment:
Are we asking too much of the suffixes here? We shouldn't be inferring the process by which we create the colours from the variable names. Maybe we should add a short guide on mixing colours to the Designer's Guide.

Using `--light` and `--dark` is certainly simpler, but it can lead to weird names like `color--dark-blue--dark`.